src/renderer/components/+workloads-pods/pod-details-container.tsx

```diff
@@ -54,6 +55,15 @@ export class PodDetailsContainer extends React.Component<Props> {
           </span>
         </DrawerItem>
         }
+        {status &&
+        <DrawerItem name={<Trans>Last Status</Trans>}>
+          <span>
+            {lastState ? `${lastState}, ` : ""}
+            {lastState === 'terminated' ? `${status.lastState.terminated.reason} (${_i18n._(t`exit code`)}: ${status.lastState.terminated.exitCode}), 
+              ${_i18n._(t`started at`)}: ${status.lastState.terminated.startedAt}, ${_i18n._(t`finished at`)}: ${status.lastState.terminated.finishedAt}` : ''}
+          </span>
+        </DrawerItem>
+        }
```

Comment:
This is a decently complicated inline render. I personally would like to see it moved to its own function

Reply:
That could work, I would have the drawer item in the separate function but that is more of a personal preference.

Reply:
I can't see any other examples where we do have the DrawerItem in the function so will leave as currently committed.